client/src/components/ViewIndividualBlogModal.jsx

```diff
@@ -0,0 +1,10 @@
+
+
+function ViewIndividualBlogModal () {
+
+// user will click on icon and see modal with all info from blogpost table in postgres
+//will I need to call an instace of this component within another component?
+
+}
+
+export default ViewIndividualBlogModal
```

Comment:
best practice is to leave an empty line as the very last line of the file. Consider adding an empty line after this line

Reply:
Thank you!